fineract-core/src/main/java/org/apache/fineract/infrastructure/core/service/database/DatabaseTypeResolver.java

```diff
@@ -58,10 +58,10 @@ public DatabaseType databaseType() {
     }
 
     public boolean isPostgreSQL() {
-        return DatabaseType.POSTGRESQL.equals(currentDatabaseType.get());
+        return DatabaseType.POSTGRESQL == currentDatabaseType.get();
```

Comment:
Why was this necessary? It's not an issue, I'm just curious.

Reply:
I would dump these functions too (see my previous comment on DatabaseType). What are we saving here? Again, let's say we add support for 10 more databases...

Reply:
I guess my point is: you can iterate over all values of an enum or use it for example in a switch statement... those "is..." functions you can only use in (ugly) if-then-else-if statements.... I think this style has a tendency to bloat code...

Reply:
These isX functions are not new and sometimes really useful, when you do not want to know which enum(s) belong to a category.
For identity please see my previous comment.